server/utils/caching/redisClient.js

```diff
@@ -45,6 +43,7 @@ class RedisCachingStrategy {
   }
 
   async get(key) {
+    logger.info(`Getting cache for '${key}'.`);
```

Comment:
this is going generate a lot of logs, it shouldn't be necessary as you should be able to see the redis calls in app insights as `dependencies`